src/Token/Token.php

```diff
@@ -132,4 +132,9 @@ public function getRefreshToken()
     {
         return $this->refreshToken;
     }
+
+    public function invalidate()
```

Comment:
If we invalidate the token setting its expire date to zero, we get problems in the construct-method and when converting the object into array.